autogluon/utils/tabular/ml/trainer/abstract_trainer.py

```diff
@@ -519,28 +268,51 @@ def train_multi_levels(self, X_train, y_train, X_val, y_val, models: List[Abstra
                 time_limit_for_level = (time_limit - (time_train_level_start - time_train_start)) / (level_end + 1 - level)
                 time_limit_core = time_limit_for_level
                 time_limit_aux = max(time_limit_for_level * 0.1, min(time_limit, 360))  # Allows aux to go over time_limit, but only by a small amount
-            if level == 0:
-                self.stack_new_level(X=X_train, y=y_train, X_val=X_val, y_val=y_val, models=models, level=level, hyperparameter_tune=hyperparameter_tune, feature_prune=feature_prune, time_limit_core=time_limit_core, time_limit_aux=time_limit_aux)
-            else:
-                self.stack_new_level(X=X_train, y=y_train, X_val=X_val, y_val=y_val, level=level, time_limit_core=time_limit_core, time_limit_aux=time_limit_aux)
-
+            if level != 0:
+                hyperparameter_tune = False  # TODO: Enable HPO on levels > 0
```

Comment:
maybe print low-priority logging statement to clarify no HPO in higher-level models?

Reply:
added